applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -246,7 +246,7 @@ Vector& ElasticIsotropic3D::CalculateValue(
         r_flags.Set( ConstitutiveLaw::COMPUTE_STRESS, true );
 
         ElasticIsotropic3D::CalculateMaterialResponsePK2(rParameterValues);
-        noalias(rValue) = rParameterValues.GetStressVector();
+        noalias(rValue) = static_cast<Vector>(rParameterValues.GetStressVector());
```

Comment:
this is not correct... beware, ou cannot statically cast an array_1d to a Vector!

Reply:
I meant this for for example

Reply:
you are right i forgot about this one